common/src/main/java/com/cinemamod/mcef/example/ExampleScreen.java

```diff
@@ -131,9 +127,9 @@ public boolean mouseDragged(double mouseX, double mouseY, int button, double dra
     }
 
     @Override
-    public boolean mouseScrolled(double mouseX, double mouseY, double delta) {
-        browser.sendMouseWheel(mouseX(mouseX), mouseY(mouseY), delta, 0);
-        return super.mouseScrolled(mouseX, mouseY, delta);
+    public boolean mouseScrolled(double mouseX, double mouseY, double scrollX, double scrollY) {
+        browser.sendMouseWheel(mouseX(mouseX), mouseY(mouseY), scrollX, 0);
```

Comment:
We should pass `scrollY` (vertical scroll) into the browser, not `scrollX` (horizontal scroll)